Dockerfile

```diff
@@ -0,0 +1,7 @@
+from node
+RUN mkdir /app
+WORKDIR /app
+COPY package.json /app/package.json
```

Comment:
We also need to copy `public` and `src`. The `Dockerfile` needs to contain everything we need to run the server without mounting local volumes/directories.

(sorry this wasn't part of my review)